### PR TITLE
Feature/kitosudv 2342 task 2344

### PR DESCRIPTION
**Pull request by mrjsawdk**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Presentation.Web/Controllers/API/V2/External/ItSystems/Mapping/IItSystemV2WriteModelMapper.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using Core.ApplicationServices.Model.System;
+using Presentation.Web.Models.API.V2.Request.System;
+
+namespace Presentation.Web.Controllers.API.V2.External.ItSystems.Mapping
+{
+    public interface IItSystemV2WriteModelMapper
```
Comment (JacobMalling) [inline Presentation.Web/Controllers/API/V2/External/ItSystems/Mapping/IItSystemV2WriteModelMapper.cs:6]: De andre mappers har ikke V2 i navnet

File: Presentation.Web/Controllers/API/V2/External/ItSystems/Mapping/ItSystemV2WriteModelMapper.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+using Core.ApplicationServices.Extensions;
+using Core.ApplicationServices.Model.Shared;
+using Core.ApplicationServices.Model.System;
+using Presentation.Web.Controllers.API.V2.External.Generic;
+using Presentation.Web.Infrastructure.Model.Request;
+using Presentation.Web.Models.API.V2.Request.System;
+
+namespace Presentation.Web.Controllers.API.V2.External.ItSystems.Mapping
+{
+    public class ItSystemV2WriteModelMapper : WriteModelMapperBase, IItSystemV2WriteModelMapper
```
Comment (JacobMalling) [inline Presentation.Web/Controllers/API/V2/External/ItSystems/Mapping/ItSystemV2WriteModelMapper.cs:12]: V2 navn

**Review by JacobMalling — APPROVED** [review]
Lille navne ting. Ellers ser det fint ud

File: Presentation.Web/Controllers/API/V2/External/ItSystems/Mapping/IItSystemV2WriteModelMapper.cs (hunk 1 above)
Comment (mrjsawdk) [reply Presentation.Web/Controllers/API/V2/External/ItSystems/Mapping/IItSystemV2WriteModelMapper.cs:6]: oh nej det har du ret i

**Pull request merged**